.pre-commit-config.yaml

```diff
@@ -1,36 +0,0 @@
-default_language_version:
```

Comment:
Please don't remove this file. To disable pre-commit locally, you can just run something like `pre-commit uninstall`